src/cli/aspencli.py

```diff
@@ -211,6 +216,22 @@ def download_samples(ctx, sample_ids):
     print(resp.headers)
     print(resp.text)
 
+@cli.group()
+def phylo_run():
+    pass
+
+@phylo_run.command(name="start")
+@click.option("-n","--name", required=True, type=str)
+@click.option("-t","--type", "tree_type", required=True, type=click.Choice(["local", "contextual"], case_sensitive=False))
+@click.argument("sample_ids", nargs=-1)
+@click.pass_context
+def start_phylo_run(ctx, name, tree_type, sample_ids):
+    # print("Name:", name, "Tree type:", tree_type, "Sample ids:", sample_ids)
+    api_client = ctx.obj["api_client"]
+    payload = { "name": name, "tree_type": tree_type, "samples": sample_ids }
+    resp = api_client.post("/api/phylo_runs", json=payload)
+    print(resp.headers)
```

Comment:
do you still need these print statements?

Reply:
nope!

Reply:
yes, this is part of the CLI so we need to let the end-user (us) know what the response was.

Reply:
we should probably add some sort of --debug flag to en/disable displaying headers. Since the CLI is currently mainly used for debugging it's probably not an urgent fix.